launcher/tests/unit/layers/InstallOption.spec.js

```diff
@@ -0,0 +1,9 @@
+import { mount } from "@vue/test-utils";
+import InstallOption from "../../../src/components/layers/InstallOption";
+describe("InstallOption.vue", () => {
+  test("renders a router-link tag with to temp.url", () => {
+    const wrapper = mount(InstallOption);
+
+    expect(wrapper.find("router-link").exists()).toBe(true);
+  });
+});
```

Comment:
You might want to check if the list in `Installoption.vue` gets rendered because it contains images which are extern resources and can cause errors (could not find file ... etc). I hope you know what i mean 😅

Reply:
Of course we can.As stefan said,at the moment we only need totally basic tests.

Reply:
Well done bud.